src/vm/i386/asmhelpers.S

```diff
@@ -472,10 +472,12 @@ NESTED_END CallDescrWorkerInternal, _TEXT
 #ifdef _DEBUG
 // int __fastcall HelperMethodFrameRestoreState(HelperMethodFrame*, struct MachState *)
 LEAF_ENTRY HelperMethodFrameRestoreState, _TEXT
+    #define     RETN 8
```

Comment:
Here is the full disass result (I attached it as a file as it is a bit long): [JIT_New.txt](https://github.com/dotnet/coreclr/files/647596/JIT_New.txt)